packages/fxa-content-server/app/scripts/lib/newsletters.js

```diff
@@ -14,38 +14,23 @@ const t = (msg) => msg;
 
 export default {
   CONSUMER_BETA: {
-    label: t('Test new Firefox products'),
+    label: t('Early access to test new products'),
     slug: 'test-pilot',
   },
   FIREFOX_ACCOUNTS_JOURNEY: {
     label: t('Get the latest news about Mozilla and Firefox'),
     slug: 'firefox-accounts-journey',
   },
   HEALTHY_INTERNET: {
-    label: t('Take action to keep the internet healthy'),
+    label: t('Help keep the internet healthy'),
     slug: 'take-action-for-the-internet',
   },
   ONLINE_SAFETY: {
     label: t('Be safer and smarter online'),
     slug: 'knowledge-is-power',
   },
-};
-
-export const newsletterNewCopy = {
```

Comment:
I removed this since it wasn't being used anywhere.

I was thinking of removing other unused newsletters like `ONLINE_SAFETY` and `FIREFOX_ACCOUNTS_JOURNEY` and updating auth-server's validity check for only these 3, happy to do that; I'd left them as-is since this way we've got a pool of available/valid newsletters and then we pull in the ones we want on the page 🤷‍♀️